.github/workflows/check.yaml

```diff
@@ -66,6 +66,9 @@ jobs:
 
       - name: Build
         run: npm run build
+      
+      - name: Build Component
+        run: npm build:component
```

Comment:
I would also consider type compilation `compile:types`, or do you think it is not needed as we already do `check:types` above?

Reply:
yeah, it should be enough